# No input parameters.
---
bool success # Whether the node successfuly armed or not.